app/main.py

```diff
@@ -2,0 +1,34 @@
+class Animal():
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+class Cat(Animal):
+    def __init__(self, name: str, appetite=3, is_hungry: bool = True):
+        super().__init__(name, appetite, is_hungry)
+
+    def catch_mouse(self):
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, appetite=7, is_hungry: bool = True):
```

Comment:
The `Dog` class should not accept an `appetite` parameter. It should automatically set `appetite` to 7 when calling the superclass `__init__` method.